lib/ts/PriorityQueue.h

```diff
@@ -110,6 +110,12 @@ PriorityQueue<T, Comp>::pop()
     return;
   }
 
+  // SKH - I suspect this assignment is not preserving entry indices correctly.
```

Comment:
Working on a test.  I think this isn't an issue in the current pop() test is that it is really only the erase() method that relies on the entry->index being accurate.  Would be nice to remove the index and just rely on array arithmetic, but that didn't look feasible with the Vec template.